src/renderer/features/lyrics/lyrics.tsx

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useRef, useState } from 'react';
+import isElectron from 'is-electron';
+import { ErrorBoundary } from 'react-error-boundary';
+import { ErrorFallback } from '/@/renderer/features/action-required';
+import { useCurrentServer, useCurrentSong } from '/@/renderer/store';
+import { SynchronizedLyrics } from './synchronized-lyrics';
+import { UnsynchronizedLyrics } from '/@/renderer/features/lyrics/unsynchronized-lyrics';
+import { LyricLine } from '/@/renderer/features/lyrics/lyric-line';
+import { Center, Group } from '@mantine/core';
+import { RiInformationFill } from 'react-icons/ri';
+import { TextTitle } from '/@/renderer/components';
+import { SynchronizedLyricsArray } from '/@/renderer/api/types';
+import { useSongLyrics } from '/@/renderer/features/lyrics/queries/lyric-query';
+
+const lyrics = isElectron() ? window.electron.lyrics : null;
+
+const ipc = isElectron() ? window.electron.ipc : null;
+
+// use by https://github.com/ustbhuangyi/lyric-parser
+const timeExp = /\[(\d{2,}):(\d{2})(?:\.(\d{2,3}))?]([^\n]+)\n/g;
+
+export const Lyrics = () => {
+  const currentServer = useCurrentServer();
+  const currentSong = useCurrentSong();
+
+  const [override, setOverride] = useState<string | null>(null);
+  const [source, setSource] = useState<string | null>(null);
+  const [songLyrics, setSongLyrics] = useState<SynchronizedLyricsArray | string | null>(null);
+
+  const remoteLyrics = useSongLyrics({
+    query: { songId: currentSong?.id ?? '' },
+    serverId: currentServer?.id,
+  });
+
+  const songRef = useRef<string | null>(null);
+
+  useEffect(() => {
+    lyrics?.getLyrics((_event: any, songName: string, lyricSource: string, lyric: string) => {
+      if (songName === songRef.current) {
+        setSource(lyricSource);
+        setOverride(lyric);
+      }
+    });
+
+    return () => {
+      ipc?.removeAllListeners('lyric-get');
+    };
+  }, []);
+
+  useEffect(() => {
+    if (currentSong && !currentSong.lyrics && !remoteLyrics.isLoading && !remoteLyrics.isSuccess) {
+      lyrics?.fetchLyrics(currentSong);
+    }
```

Comment:
This check is not working as expected.
In `lyric-query.ts` -- `useSongLyrics`, the `queryFn` returns null if the server is not Jellyfin.

This causes `remoteLyrics.isSuccess` to always equal true because no error was thrown in the queryFn. The condition will always be falsy and the fetchLyrics is skipped if the song is Navidrome.

Let's split the conditional to make it clearer
```typescript
    const hasTaggedLyrics = currentSong && currentSong.lyrics;
    const hasLyricsResponse =
      !remoteLyrics.isLoading && remoteLyrics?.isSuccess && remoteLyrics?.data !== null;

    if (!hasTaggedLyrics && !hasLyricsResponse) {
      lyrics?.fetchLyrics(currentSong);
    }
```